src/main/java/frc/robot/subsystems/Drive/DriveSubsystem.java

```diff
@@ -325,6 +325,6 @@ public void onLimelightDataUpdate(boolean targetVisible, double[] botPose) {
           double x = botPose[0];
           double y = botPose[1];
           double rotationRadians = botPose[2];
-          currentPose = new Pose2d(x, y, new Rotation2d(rotationRadians));
+          Pose2d updatePose = new Pose2d(x, y, new Rotation2d(rotationRadians));
```

Comment:
not what i ment you need to store it at the subsystem but just with a diffrent name because current pose is allready on usage